docker/data-science/notebooks/ch03/1.vectors-and-text-similarity.ipynb

```diff
@@ -624,9 +623,9 @@
       "\n",
       "Document TF-IDF Vector Scores\n",
       "Labels: ['the', 'cat', 'in', 'the', 'hat']\n",
-      "  doc1: [0.0915, 0.0, 0.0905, 0.0915, 0.0]\n",
-      "  doc2: [0.0617, 1.4282, 0.0557, 0.0617, 1.7983]\n",
-      "  doc3: [0.0, 1.2041, 0.0664, 0.0, 1.5161]\n",
+      " doc1: [0.0915, 0.0, 0.0905, 0.0915, 0.0]\n",
```

Comment:
I visually prefer the  previous extra space so the "[" line up, but not a big deal either way.